packages/components/src/templates/next/components/complex/ContactInformation/components/DefaultContactInformationUI.tsx

```diff
@@ -0,0 +1,128 @@
+import DOMPurify from "isomorphic-dompurify"
+
+import type { ContactInformationUIProps } from "~/interfaces"
+import { tv } from "~/lib/tv"
+import { LinkButton } from "../../../internal/LinkButton"
+import {
+  commonContactInformationStyles,
+  commonContactMethodStyles,
+} from "./common"
+import { ContactMethod, LoadingContactMethod } from "./ContactMethod"
+
+const createDefaultContactInformationStyles = tv({
+  extend: commonContactInformationStyles,
+  slots: {
+    container: "gap-9 py-12",
+    titleAndDescriptionContainer: "lg:max-w-3xl",
+    contactMethodsContainer: "md:grid md:grid-cols-2",
+    urlButtonContainer: "block",
+  },
+})
+
+const createDefaultContactMethodStyles = tv({
+  extend: commonContactMethodStyles,
+})
+
+export const DefaultContactInformationUI = ({
+  whitelistedMethods,
+  title,
+  description,
+  methods,
+  otherInformation,
+  referenceLinkHref,
+  label,
+  LinkComponent,
+  isLoading,
+  acceptHtmlTags = false,
+}: ContactInformationUIProps) => {
+  const compoundStyles = createDefaultContactInformationStyles({
+    isLoading,
+  })
+  const contactMethodStyles = createDefaultContactMethodStyles({
+    isLoading,
+  })
+
+  const filteredMethods = whitelistedMethods
+    ? methods.filter(
+        (method) => method.method && whitelistedMethods.includes(method.method),
+      )
+    : methods
+
+  const descriptionText = isLoading ? "" : (description ?? "")
+
+  return (
+    <div className={compoundStyles.container()}>
+      <div className={compoundStyles.titleAndDescriptionContainer()}>
+        {(title || isLoading) && (
+          <h2 className={compoundStyles.title()}>{isLoading ? "" : title}</h2>
+        )}
+        {(!!description || isLoading) &&
+          (acceptHtmlTags ? (
+            <p
+              className={compoundStyles.description()}
+              dangerouslySetInnerHTML={{
+                __html: DOMPurify.sanitize(descriptionText, {
+                  ALLOWED_TAGS: ["br"],
+                }),
+              }}
+            />
+          ) : (
+            <p className={compoundStyles.description()}>{descriptionText}</p>
+          ))}
```

Comment:
question: are we able to use/extend BaseParagraph for this? Ideally we consolidate to that to avoid doing `dangerouslySetInnerHTML` and also to ensure that the text styles are consistent.

(same comment applies for all other usages of `dangerouslySetInnerHTML`)